### ISSUE TITLE
Add integration and or e2e tests

### ISSUE THREAD

**Issue by jamesvclements** [bartop-io/bartop#94]
Once https://github.com/dpopp07/bartop/pull/81 is merged, we'll have our first full UI -> API -> DB flow. That means we could do a full e2e test, which will probably take a bit to set-up but would be nice to have in our CI.

1. Research e2e testing options
2. Add the first test
